other/materials_designer/specific_examples/run_bandgap_workflow.ipynb

```diff
@@ -0,0 +1,200 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    from mat3ra.wode.analyzers.electronic import KPointAnalyzer, CutoffAnalyzer, SmearingAnalyzer, BandsAnalyzer
</pre><p>These are called context providers at current, not analyzers, but the concept is similar</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/231/discussion/#comment-2176377756'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>